libs/api/src/main/java/de/cyface/api/model/Role.java

```diff
@@ -75,12 +75,16 @@ public Role(Type type, String group) {
      * @param databaseValue A role entry from the database.
      */
     public Role(final String databaseValue) {
+        final var isAdmin = databaseValue.matches(Type.ADMIN.getRegex());
         final var isGuest = databaseValue.matches(Type.GUEST.getRegex());
         final var isGroupUser = databaseValue.matches(Type.GROUP_USER.getRegex());
         final var isGroupManager = databaseValue.matches(Type.GROUP_MANAGER.getRegex());
         if (isGuest) {
             this.type = Type.GUEST;
             this.group = null;
+        } else if (isAdmin) {
+            this.type = Type.ADMIN;
+            this.group = null;
```

Comment:
Forgot that there is an (unused) "admin" role before ... which was why some tests failed. This fixes this.